pkg/google/networking/pricing_map.go

```diff
@@ -0,0 +1,190 @@
+package networking
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log/slog"
+	"strings"
+
+	"cloud.google.com/go/billing/apiv1/billingpb"
+	"github.com/grafana/cloudcost-exporter/pkg/google/client"
+)
+
+var (
+	forwardingRuleDescription        = "Forwarding Rule"
+	inboundDataProcessedDescription  = "Inbound Data Processing"
+	outboundDataProcessedDescription = "Outbound Data Processing"
+	serviceName                      = "Networking"
+	ResourceGroup                    = "LoadBalancing"
+
+	ErrRefreshingPricingMap            = errors.New("error refreshing pricing map")
+	ErrGettingNetworkingServiceName    = errors.New("error getting networking service name")
+	ErrNoSKUsFoundForNetworkingService = errors.New("no skus found for networking service")
+	ErrParsingSKUs                     = errors.New("error parsing skus")
+	ErrRegionNotFound                  = errors.New("region not found")
+	ErrUnknownDescription              = errors.New("unknown description")
+)
+
+type ParsedSkuData struct {
+	Region      string
+	Price       float64
+	Description string
+}
+
+func NewParsedSkuData(region string, price float64, description string) *ParsedSkuData {
+	return &ParsedSkuData{
+		Region:      region,
+		Price:       price,
+		Description: description,
+	}
+}
+
+type pricing struct {
+	forwardingRuleCost        float64
+	inboundDataProcessedCost  float64
+	outboundDataProcessedCost float64
+}
+
+func NewPricing() *pricing {
+	return &pricing{
+		forwardingRuleCost:        0,
+		inboundDataProcessedCost:  0,
+		outboundDataProcessedCost: 0,
+	}
+}
+
+type pricingMap struct {
+	pricing   map[string]*pricing
+	logger    *slog.Logger
+	gcpClient client.Client
```

Comment:
since we have multiple goroutines reading/writing to the pricingMap, I think we do want a mutex managing it
```suggestion
	gcpClient client.Client
	mu sync.RWMutex
```